Modules/Entities/PhoenixDocument/Sources/PhoenixDocument/Component/ProjectConfiguration.swift

```diff
@@ -39,6 +43,8 @@ public struct ProjectConfiguration: Codable, Hashable {
         macrosFolderName = try container.decodeIfPresent(String.self, forKey: .macrosFolderName) ?? ProjectConfigurationConstants.defaultMacrosFolderName
         swiftVersion = (try? container.decode(String.self, forKey: .swiftVersion)) ?? "5.9"
         defaultOrganizationIdentifier = try? container.decodeIfPresent(String.self, forKey: .defaultOrganizationIdentifier)
+        platforms = (try? container.decode(Component.Platforms.self, forKey: .platforms)) ?? .empty
+        print(platforms)
```

Comment:
You forgot this print statement here.

Reply:
Resolved in f0f74230e63634ac7f4cef39889488c45a0fbd6c